.github/workflows/docker-publish.yml

```diff
@@ -88,19 +89,16 @@ jobs:
       uses: docker/setup-qemu-action@v3
     - name: Test image
       run: |
-        ARCH=$(cut -d "/" -f2 <<< ${{ matrix.docker-platform }})
         source .venv/bin/activate
-        if [ "${{ matrix.docker-platform }}" == "linux/arm64" ]; then
-          DOCKER_PLATFORM="${{ matrix.docker-platform }}" DOCKER_IMAGE="$DOCKER_BUILD_REPOSITORY:$ARCH-$SHORT_SHA" \
+        DOCKER_IMAGE="$DOCKER_BUILD_REPOSITORY:${{ matrix.arch }}-$SHORT_SHA" \
+        if [ "$DOCKER_PLATFORM" == "linux/arm64" ]; then
```

Comment:
DOCKER_PLATFORM env variable is already set, so no need to provide it here. If you expect being explicit, please let me know